src/pps/pps.proto

```diff
@@ -364,12 +362,15 @@ enum PipelineState {
 // PFS (and, by pointing to a PFS commit, de facto tracks the pipeline's
 // version)
 message StoredPipelineInfo {
-  PipelineState state = 1;
-  string reason = 2;
-  pfs_v2.Commit spec_commit = 3;
-  map<int32, int32> job_counts = 4;
-  string auth_token = 5;
-  JobState last_job_state = 6;
+  Pipeline pipeline = 1;
+  uint64 version = 2;
+  pfs_v2.Commit original_spec_commit = 3; // The first spec commit for this version of the pipeline
```

Comment:
Can a pipeline version have multiple spec commits? I thought the purpose of using the fileset functionality for the spec commit was to make it transactional such that there is only one spec commit for a version. If that is the case, then we could probably just have this be `spec_commit`. Maybe this is needed for aliases?

Reply:
So I removed the part in `StartPipeline` and `StopPipeline` where they would create new spec commits, but alias commits are technically commits and we should have plenty of those in the spec repo - so this records the original spec commit from when the pipeline was updated or created.

Reply:
Okay, it does seem a bit confusing though. I'm not sure what the benefit of the "original" part is. Do we ever need to consider the original vs alias commits, or do we just grab the spec commit when we need a commit that has the correct spec file?